conda/recipes/cuvs/meta.yaml

```diff
@@ -54,10 +54,8 @@ requirements:
     - pylibraft {{ minor_version }}
     - libcuvs {{ version }}
     - python x.x
-    - rmm ={{ minor_version }}
```

Comment:
`rmm` (Python) is not used directly in `cuvs` Python code. It is not needed at build or run time.